client/blocks/importer/wordpress/import-everything/pre-migration/style.scss

```diff
@@ -8,6 +8,16 @@
 	padding: 0 rem(20px);
 	max-width: 100%;
 
+	.import__heading {
+		margin-bottom: 0;
+		.onboarding-title {
+			line-height: 57.6px;
+			letter-spacing: -0.32px;
+			font-size: 3rem;
+			margin-bottom: 8px;
+		}
+	}
+
```

Comment:
We should not override the `.onboarding-title` specifically for this page. We use the `Title` and `SubTitle` throughout the onboarding flow, and recently, we [had PR clean up the style overrides](https://github.com/Automattic/wp-calypso/commit/1b6915485b7a5eaf88a4d838ba8cc730647690db) to be consistent between the steps/screens.

If we want to change something, I would suggest doing it in one place and applying it globally.

Reply:
Regression:

<img width="1117" alt="Screenshot 2024-01-31 at 11 28 56" src="https://github.com/Automattic/wp-calypso/assets/1241413/78cbe16c-260e-46f9-b6db-db0349b0124a">

Reply:
Thanks for the review @bogiii 

I'll revert these styling changes and keep the existing styles.